pubtools/pulplib/_impl/model/repository/base.py

```diff
@@ -58,6 +58,13 @@ class PublishOptions(object):
     :meth:`~pubtools.pulplib.Distributor.is_rsync` is ``True``.
     """
 
+    rsync_extra_args = pulp_attrib(default=None, type=list)
```

Comment:
Could you please add converter=frozenlist here to make it immutable?  PublishOptions instances are currently immutable, but this will be lost if we allow a plain mutable list here.